server/routerlicious/package.json

```diff
@@ -138,11 +138,11 @@
 	},
 	"pnpm": {
 		"commentsOverrides": [
-			"@types/node is overridden to v18 to ensure all packages are building using the v18 types.",
+			"@types/node is overridden to v20 to ensure all packages are building using the v20 types, matching the version of note required according to the readme. This also fixes a compile error due to conflicting overloads from multiple versions of the globals in @types/node/module.d.ts",
 			"oclif includes some AWS-related features, but we don't use them, so we override those dependencies with empty packages. This helps reduce lockfile churn since the deps release very frequently."
 		],
 		"overrides": {
-			"@types/node@<18": "^18.19.39",
+			"@types/node": "~20.0.0",
```

Comment:
Without this change, the build was filing with a compile error from conflicting global.

I made this specific change to align with the readme's required node version.

Annoyingly this doesn't seem to fully work as the lock file still has another version in it, but it fixed it enough for the build to work.